handlers/handlers.go

```diff
@@ -0,0 +1,111 @@
+package handlers
+
+import (
+	"encoding/csv"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"os"
+	"strconv"
+
+	"github.com/gorilla/mux"
+)
+
+func GetAll(w http.ResponseWriter, r *http.Request) {
+	pokemonMap := make(map[int]string)
+	w.Header().Set("Content-Type", "application/json")
+	w.WriteHeader(http.StatusOK)
+
+	file, err := os.Open("assets/pokemon.csv")
+	if err != nil {
+		fmt.Println(err)
+	}
+	reader := csv.NewReader(file)
+	records, _ := reader.ReadAll()
+
+	for _, pokemon := range records {
+		index, _ := strconv.Atoi(pokemon[0])
+		pokemonMap[index] = pokemon[1]
+	}
+	response, _ := json.Marshal(pokemonMap)
+	w.Write(response)
+}
+
+func GetById(w http.ResponseWriter, r *http.Request) {
+	pathParams := mux.Vars(r)
+	pokemonMap := make(map[string]string)
+	pokemonId := pathParams["pokemonId"]
+	w.Header().Set("Content-Type", "application/json")
+
+	file, err := os.Open("assets/pokemon.csv")
+	if err != nil {
+		fmt.Println(err)
```

Comment:
Instead of printing the error, we can log it. Take a look at [logrus](https://github.com/sirupsen/logrus), for example.